src/server/logging/log_format_string.js

```diff
@@ -23,18 +23,18 @@ const typeColors = {
   info: 'green',
   error: 'red',
   warning: 'red',
-  fatal: 'magenta',
-  status: 'yellow',
-  debug: 'brightBlack',
-  server: 'brightBlack',
+  fatal: 'magentaBright',
+  status: 'yellowBright',
+  debug: 'gray',
+  server: 'gray',
   optmzr: 'white',
```

Comment:
If you go with `dim` instead of `gray` you can use `reset` instead of `white` so that when the text is rendered on a white background like on jenkins it will be visible.

Reply:
Can you provide example code?

Reply:
I'm just suggesting that if you wanted to switch to non-specific versions of white/gray/black you could use `optmzr: 'reset',` instead of `optmzr: 'white',`.

Reply:
when i do that, i get this, where the grays are gone:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/13165182/34059438-a9c2c82e-e1a4-11e7-832c-f5539c331c7f.png">
Here are the changes I made:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/13165182/34059454-bc3e6f08-e1a4-11e7-872d-95441c2b7a78.png">